doc/metrics.md

```diff
@@ -0,0 +1,65 @@
+# Instance metrics
+LXD provides metrics for all running instances.
+These are useful for monitoring instances.
```

Comment:
`LXD provides metrics for all running instances. Those covers CPU, memory, network, disk and process usage and are meant to be consumed by Prometheus and likely graphed in Grafana.`

Reply:
Should also add that `In cluster environments, LXD will only return the values for instances running on the server being accessed. It's expected that each cluster member will be scraped separately.`